package.json

```diff
@@ -44,6 +60,7 @@
   ],
   "dependencies": {
     "body": "^5.1.0",
+    "cross-env": "^5.1.3",
```

Comment:
If I'm not mistaken, cross-env should go devDependencies.

Reply:
@n-riesco I am inclined to keep cross-env in `dependencies{}` but I could easily move it to `devDependencies{}` if you'd like.

The thing is, this software is going to be executed by end-users in the command line (if only as a subprocess in a Pip package, or maybe more directly) on Windows and OS X environments. I worry about consistency across all environments with regards to environment variables.

@n-riesco et al, what do you think?

Reply:
@JamesCropcho That's not my understanding. We want to distribute `image-exporter` as an electron app, so that end-users don't have to install `node`.

If we include `cross-env` in `dependecies`, then `cross-env` will be included in the electron app (and the electron app doesn't use it).

Reply:
I have moved cross-env to `devDependencies`.